lib/create-plugin.js

```diff
@@ -406,35 +410,24 @@ function createModuleUsageMetric(agent) {
   ).incrementCallCount()
 }
 
-function isIntrospectionQuery(operation, captureIntrospectionQueries) {
-  if (captureIntrospectionQueries) {
-    return false
-  }
-
+function isIntrospectionQuery(operation) {
   return operation.selectionSet.selections.every((selection) => {
     const fieldName = selection.name.value
     return INTROSPECTION_TYPES.includes(fieldName)
   })
 }
 
-function isServiceDefinitionQuery(operation, captureServiceDefinitionQuery) {
-  if (captureServiceDefinitionQuery) {
-    return false
-  }
-
+function isServiceDefinitionQuery(operation) {
   return operation.name && operation.name.value === SERVICE_DEFINITION_QUERY_NAME
 }
 
-function isHealthCheckQuery(operation, captureHealthCheckQuery) {
-  if (captureHealthCheckQuery) {
-    return false
-  }
-
+function isHealthCheckQuery(operation) {
   return operation.name && operation.name.value === HEALTH_CHECK_QUERY_NAME
 }
 
 function shouldIgnoreTransaction(operation, config, logger) {
   if (
+    !config.captureIntrospectionQueries &&
```

Comment:
looks like you check config in if and no longer need to pass it into every is<type> function

Reply:
Good call! Forgot to remove those. 🪓